SlackClone/SlackMessageOptionsResolver.swift

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatUI
+import UIKit
+
+final class SlackMessageOptionsResolver: ChatMessageLayoutOptionsResolver {
+    override func optionsForMessage(
+        at indexPath: IndexPath,
+        in channel: ChatChannel,
+        with messages: AnyRandomAccessCollection<ChatMessage>
+    ) -> ChatMessageLayoutOptions {
+        var options = super.optionsForMessage(at: indexPath, in: channel, with: messages)
+        options.remove([.flipped, .bubble, .metadata, .avatar, .avatarSizePadding, .authorName, .threadInfo, .reactions])
```

Comment:
This is one of the demo apps so we don't care about it ATM, however, I think this brings up an interesting topic:

With such dramatic changes in the default options, I'd rather provide yet another implementation instead of updating the `super` one. 

But that's not possible currently because of `ChatMessage+Extensions` being internal 🤔